dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -596,7 +596,7 @@ public void withdrawPatchTest() throws Exception {
         // is used in the provenance note.
         String token = getAuthToken(admin.getEmail(), password);
 
-        List<Operation> ops = new ArrayList<Operation>();
+        List<Operation> ops = new ArrayList<>();
```

Comment:
I was surprised to see a number of these type of changes. Is this refactoring from the IDE?

Reply:
NetBeans flags them and I usually tell it "yes, fix that" to get rid of the clutter of yellow flags obscuring issues that can't be mechanically fixed.  Our code is full of archaisms and I tend to tidy them up as I go.

NetBeans also notices things like "you could use try-with-resources," "you could use `isEmpty()` instead of `size() <= 0," "you could use an enhanced `for` loop instead of fiddling with index variables" and the like, and I often let it have its way.  It's never been wrong or broken any code, and the result is more understandable and less error-prone.

I think of these little improvements as "cleaning up the workbench" before tackling the main task.

Reply:
Great, I'm not a regular Java user so I don't know which format is archaic, but I believe you! I am happy to have those cleanups as well, and suspected it might be that thanks to @mwoodiupui's notes in the PR.